src/main/java/com/amazon/opendistroforelasticsearch/security/DefaultObjectMapper.java

```diff
@@ -61,6 +63,22 @@
         YAML_MAPPER.enable(JsonParser.Feature.STRICT_DUPLICATE_DETECTION);
     }
 
+    public static void inject(final InjectableValues.Std injectableValues) {
+        objectMapper.setInjectableValues(injectableValues);
+        YAML_MAPPER.setInjectableValues(injectableValues);
+        defaulOmittingObjectMapper.setInjectableValues(injectableValues);
+    }
+
+    public static boolean getOrDefault(Map<String, Object> properties, String key, boolean defaultValue) {
+        Boolean value = (Boolean)properties.get(key);
```

Comment:
can we directly cast as boolean?

Reply:
It is of type Object in properties

Reply:
No, it will cause `NullPointerException` if the property is missing in the `properties`. Should `properties` always contain `key` property, it would be possible to cast directly to `boolean` (unboxing).